pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -297,27 +297,23 @@ func TestGetNewRC(t *testing.T) {
 	}
```

Comment:
While here can you also rename this test from `TestGetNewRC` to `TestGetNewRS`? Thanks